src/app/main/component/events/components/create-edit-events/create-edit-events.component.html

```diff
@@ -7,7 +7,8 @@
     <h3 *ngIf="!editMode">Create Event</h3>
     <h3 *ngIf="editMode">Edit Event</h3>
     <p>
-      Please provide as much details as you can - place and time of event, the goal of gathering etc. You can come back and update event
+      Please provide as much details as you can - place and time of event, the goal of gathering etc. You can come back
+      and update event
```

Comment:
Shouldn't this text be translated?

Reply:
Yes. It should to do. But it is another issue. There are requirements of task:
https://github.com/ita-social-projects/GreenCity/issues/4604